packages/base/src/index.js

```diff
@@ -71,6 +71,28 @@ module.exports = {
     'func-name-matching': 'error',
     'grouped-accessor-pairs': 'error',
     'guard-for-in': 'error',
+    'id-denylist': [
+      // This sets this rule to 'error', the rest are the forbidden IDs.
+      'error',
+      // These are basically all useless contractions.
+      'buf',
+      'err',
+      'cb',
+      'cfg',
+      'hex',
+      'msg',
+      'num',
+      'opt',
+      'str',
+    ],
+    'id-length': [
+      'error',
+      {
+        min: 2,
+        properties: 'never',
+        exceptionPatterns: ['[i-k]', '[x-z]', '[r-v]'],
```

Comment:
What do these exceptions correspond to? :thinking:

Reply:
Ah, they correspond to: `i, j, k`, `x, y, z`, and `r, s, v`.

It might be simpler just to spell it out, and perhaps also only do `i, j, k`. The motivator there traditional `for` loops. `r, s, v` is of course used in `eth-sig-util`, but I'm realizing we can create exceptions for those cases.

Reply:
Yeah, I switched it to `i, j, k`.